### DISCUSSION TITLE
what the line color/brightness in the git blame means?

### DISCUSSION THREAD

**Discussion by ghost** [dandavison/delta#1682] [Q&A] [answered]
![image](https://github.com/dandavison/delta/assets/72302817/4ff6b3b0-a028-4d99-84fa-603c3b95147a)
here you can see that some lines are brighter than others, that it means?, thanks!

**Comment by dandavison** [comment, marked as answer]
Hi, each block of color identifies adjacent lines from the same commit.

**Reply by ghost** [reply]
i knew it was something like that, but my sleepy brain couldn't handle it lol
